docs/pages/client/sdk/unity/important-concepts.md

```diff
@@ -0,0 +1,107 @@
+# Dojo Unity Concepts
+
+To dive into the exciting world of on-chain games and worlds with Unity, let's explore some essential concepts:
+
+## World Manager
+
+The World Manager is the central hub for organizing and controlling entities within your Dojo world in Unity.
+
+![world-manager](/unity/world-manager.png)
+
+During initialization, the World Manager receives WorldManagerData, which defines essential settings like your Torii URL, RPC URL, and world address. While these settings are initially provided, you have the flexibility to adjust them by creating different scriptable objects.
+
+![world-manager-data](/unity/world-manager-data.png)
+
+In Unity, entities are represented by GameObject instances. The World Manager simplifies their management by offering methods to:
+
+* Add and remove entities
+* Access entities by name or list all entities
+
+## Synchronization Master
+
+The Synchronization Master acts as the bridge between Unity and your Dojo world, seamlessly synchronizing and managing entities.
+
+![sync-master](/unity/sync-master.png)
+
+Key Features:
+
+* Control synchronization: Set the maximum number of entities to synchronize.
+* Event-driven communication:
+  * OnSynchronized: Notifies you when entities were successfully synchronized from Dojo world to Unity.
+  * OnEntitySpawned: Triggered whenever a new entity is spawned in the Unity environment.
+
+* Dynamic entity management:
+  * SynchronizeEntities: Asynchronously retrieves and spawns entities from the Dojo world in the Unity environment.
+  * HandleEntityUpdate: Dynamically updates existing entities or spawns new ones based on changes received from the Dojo world, ensuring seamless synchronization.
+
+## Models
```

Comment:
users dont need to define models themselves so maybe highlight that models can be generated using sozo build --unity